fe/fe-core/src/main/java/org/apache/doris/statistics/HMSAnalysisTask.java

```diff
@@ -67,10 +70,8 @@ public class HMSAnalysisTask extends BaseAnalysisTask {
             + "NOW() "
             + "FROM `${catalogName}`.`${dbName}`.`${tblName}`";
 
-    private static final String ANALYZE_SQL_PARTITION_TEMPLATE = "INSERT INTO "
-            + "${internalDB}.${columnStatTbl}"
-            + " SELECT "
-            + "CONCAT(${tblId}, '-', ${idxId}, '-', '${colId}') AS id, "
+    private static final String ANALYZE_PARTITION_TEMPLATE = " SELECT "
+            + "CONCAT(${tblId}, '-', ${idxId}, '-', '${colId}', '-', ${partId}) AS id, "
```

Comment:
And this makes the template not like `ANALYZE_TABLE_TEMPLATE`, which has `insert into`

Reply:
This is for execute multiple partitions in one sql. Will batch insert after collect multiple partitions stats. This could reduce the IO pressure for Doris